src/main/java/seedu/address/ui/EventTaskListCard.java

```diff
@@ -34,7 +35,7 @@ public EventTaskListCard(EventTask eventTask, int index) {
             nameLabel.setText(eventTask.getName().toString());
             startLabel.setText(eventTask.getStart().toString());
             endLabel.setText(eventTask.getEnd().toString());
-            if (eventTask.getEnd().isBefore(LocalDateTime.now())) {
+            if (!(new EventNotInPastPredicate()).test(eventTask)) {
```

Comment:
Right, I didn't read the code carefully, but my point still stands. Call it `EventUnfinishedPredicate` instead.

But anyway, this comment is obsolete. My #162 combines the individual predicates into one single `TaskPredicate` interface, which greatly simplifies all the predicate-fudging-around. Maybe I could merge it now?